# corelationsRNA

### Paper summary

In the course of the research, virus signatures for SARS-CoV-2 and MERS-CoV were discovered, highlighting previously unexplored distant correlations within their genomes. These signatures encompass the memory function of a random system and a matrix of pairwise correlation functions. They offer a numerical representation of the far-reaching mutual impact of nucleotides, spanning the entire genomic sequence. Although largely uninvestigated until now, these distant correlations, along with the more commonly examined local correlations, substantially influence the virus's properties and its genome's regulatory functions.

Furthermore, the study introduced a methodology that can help with tasks such as evaluating the extent of differences between viruses based on specific parameters, classifying DNA and RNA fragments, and quickly detecting health hazards, among other applications. The research anticipates comparisons between the SARS-CoV-2 signature and those of other coronaviruses in relation to their biological and clinical outcomes, as well as cross-referencing signatures of various viruses.

Consequently, the findings from this study will yield exclusive, additional insights into the structure of viruses, potentially benefiting virologists, clinicians, and epidemiologists in their future work.

## Usage
### input: 
.fasta rna sequence file (you can get it on ncbi)

## Results
### See paragraph 4 for results for MERS-CoV and SARS-CoV-1
![image](https://user-images.githubusercontent.com/36309634/233639817-a6c7672c-9e36-4625-808c-8f3baa986c55.png)
#### Correlation function for SARS-CoV-2
![image](https://user-images.githubusercontent.com/36309634/233640004-f021e698-091e-4544-8558-1b3def15dff4.png)
#### Variety function for SARS-CoV-2
![image](https://user-images.githubusercontent.com/36309634/233640080-d67a89e0-20d7-48d6-9402-cdff5aa0ba32.png)
#### Memory function for SARS-CoV-2


